### ISSUE TITLE
 Final products to resolve

### ISSUE THREAD

**Issue by kimrutherford** [japonicusdb/japonicus-curation#13]
Add products to remaining proteins

**Comment by ValWood** [comment]
We don't need to do this,  at least yet. 
I will squeeze out all of the better description first.

**Comment by ValWood** [comment]
[main 82f94f2]
added
SJAG_02983              dubious
SJAG_06595              dubious
SJAG_06420              dubious
SJAG_06013              dubious
SJAG_05617              dubious
SJAG_06600              dubious
SJAG_04254              dubious
SJAG_00121              dubious
SJAG_01904              dubious
SJAG_03610              dubious
SJAG_06616      pll1    PLLLP motif protein
SJAG_04834      pll2    PLLLP motif protein
SJAG_06615      pll3    PLLLP motif protein
SJAG_04804      pll4    PLLLP motif protein
SJAG_04801      pll5    PLLLP motif protein
SJAG_02677      pll6    PLLLP motif protein
SJAG_02675      pll7    PLLLP motif protein
SJAG_04798      pll8    PLLLP motif protein
SJAG_04382      pll9    PLLLP motif protein
SJAG_00026      pll10   PLLLP motif protein
SJAG_03607      pll11   PLLLP motif protein
SJAG_02108      pll12   PLLLP motif protein
SJAG_04796      pll13   PLLLP motif protein
SJAG_00780      pll14   PLLLP motif protein
SJAG_00709      pll15   PLLLP motif protein

@snezhkaoliferenko  that is a big japoonicus specific gene family!

I am assigning some as dubious, they don't look real to me. I have notes for why I made the call (usually, no syntenic ortholog in syntenic region in. pombe, low complexity sequence, no hits with jackhmmr or PSI-blast, small orfs).

More later...

**Comment by ValWood** [comment]
some more poroducs added
[main 2083134] more products
SJAG_06352              dubious
SJAG_04976              dubious
SJAG_06630              dubious
SJAG_05255              dubious
SJAG_05889              dubious
SJAG_04075              S. Japonicas specific protein 
SJAG_06631      wcy1    japonicas specific protein family with conserved tryptophans cysteines and tyrosines 
SJAG_05729      wcy2    japonicas specific protein family with conserved tryptophans cysteines and tyrosines 
SJAG_05746      wcy3    japonicas specific protein family with conserved tryptophans cysteines and tyrosines 
SJAG_05745      wcy4    japonicas specific protein family with conserved tryptophans cysteines and tyrosines 
SJAG_04247              transmembrane protein GPR107/GPR108-like

**Comment by ValWood** [comment]
- [ ] FIX spelling japonicas-> japonicus

**Comment by ValWood** [comment]
added new products 
SJAG_03134	fungal conserved protein YML020W protein ortholog, PTHR47349 family
SJAG_02710	Schizosaccharomyces specific family with conserved tryptophan
SJAG_05920	Schizosaccharomyces specific family with conserved tryptophan 
SJAG_03148	Schizosaccharomyces japonicus specific protein with predicted signal peptide
SJAG_04045	Schizosaccharomyces japonicus specific protein
SJAG_02195	Schizosaccharomyces japonicus specific protein
SJAG_00351	Schizosaccharomyces japonicus specific protein
SJAG_02159	Schizosaccharomyces japonicus specific protein
SJAG_00672	Schizosaccharomyces japonicus specific protein
SJAG_03439	Schizosaccharomyces japonicus disordered protein
SJAG_01158	Schizosaccharomyces japonicus disordered protein
SJAG_00528	Schizosaccharomyces japonicus disordered protein
SJAG_02122	Schizosaccharomyces japonicus specific low-complexity Glycine rich protein
SJAG_00194	Schizosaccharomyces japonicus specific protein
SJAG_01757    Schizosaccharomyces japonicus protein family, in between Wcy 
SJAG_01689    Schizosaccharomyces japonicus protein family, in between Wcy 
SJAG_06637	transposon fragment
SJAG_02928	transposon fragment
SJAG_03765	transposon fragment
SJAG_05352	transposon fragment
SJAG_03766	transposon fragment
SJAG_06640	ReqQ-type subtelomeric helicase fragment
SJAG_02967	dubious
SJAG_02966	dubious
SJAG_00927	dubious
SJAG_00929	dubious
SJAG_05336	dubious
SJAG_00994 	dubious
SJAG_06212	dubious
SJAG_06471	dubious
SJAG_00862	dubious
SJAG_03885	dubious
SJAG_03390	dubious


[main 726e4ae]

**Comment by ValWood** [comment]
otg1	alpha-1,3-galactosyltransferase Otg2
fixed to otg3 (syntenic position). This means there is no otg1.

**Comment by ValWood** [comment]
SJAG_05118 ->  changed to 
transposon related zinc Knuckle and gag-polyprotein aspartyl protease

**Comment by ValWood** [comment]
Product lines added.

For these: I could not unequivocally pronounce the orthologs yet as  I have no alignments (Fasta, jackhmmr and PSI-blast failed to reveal anything. Usually, these are in synteny breakpoints which means they are only syntenic to one direction, leading to the possibility that this is an unrelated gene. Usually, they look reasonable  in length and composition, but even if I can find the pombe ortholog in octosporus and cryophillus I just haven't b been able to detect any conserved motifs in japonicus  as yet.  This is a longer task. There may also be gene prediction errors precluding a good alignment...
SJAG_03510		Mis19 candidate ortholog
SJAG_01815		S. pombe SPCC569.09 candidate ortholog
SJAG_05292		Dms1 candidate ortholog (possible gene prediction error)

The rest, no obvious orthologs:
SJAG_04557		Schizosaccharomyces japonicus specific protein
SJAG_04742		Schizosaccharomyces japonicus specific protein
SJAG_01525		Schizosaccharomyces japonicus specific protein some similarity to proteases
SJAG_03639		Schizosaccharomyces japonicus specific protein
SJAG_03109		dubious
SJAG_03445		Schizosaccharomyces japonicas specific protein
SJAG_03800		Schizosaccharomyces japonicus specific protein
SJAG_03201		Schizosaccharomyces japonicus specific 6/7 TM domain protein family
SJAG_03202		Schizosaccharomyces japonicus specific 6/7 TM domain protein family
SJAG_02190		Schizosaccharomyces japonicus specific 6/7 TM domain protein family
SJAG_01607		dubious
SJAG_01451		Schizosaccharomyces japonicus specific protein, possibly be part of SJAG_01450
SJAG_04985		Taz1-like protein
SJAG_06546		Schizosaccharomyces japonicus specific protein
SJAG_01893		dubious

**Comment by ValWood** [comment]
SJAG_03974	ecl1 extender of chronological lifespan protein Ecl  (is 1:many so I should have fixed this one already)

**Comment by ValWood** [comment]
SJAG_02143              Schizosaccharomyces specific protein, no S. pombe detected
SJAG_06620              Schizosaccharomyces japonicus specific protein
SJAG_00367              cell surface adhesion molecule with signal peptide (predicted)
SJAG_04630              candidate ortholog for Meikin
SJAG_04636              dubious
SJAG_03178              Schizosaccharomyces japonicus specific serine/threonine rich protein 
SJAG_02792              Schizosaccharomyces japonicus specific protein
SJAG_00454              Schizosaccharomyces japonicus specific protein

[main 9ce4388] more products (I Think?)

**Comment by ValWood** [comment]
Next bunch:

SJAG_02893		Schizosaccharomyces japonicus specific disordered protein 
SJAG_01555		Schizosaccharomyces japonicus specific disordered protein 
SJAG_03875      	Schizosaccharomyces japonicus specific disordered protein
SJAG_03645		transposable element related protein
SJAG_00006		transposable element zinc finger related protein
SJAG_01552		Schizosaccharomyces japonicus specific protein/dubious
SJAG_02925  		Schizosaccharomyces japonicus specific disordered protein
SJAG_04269		Schizosaccharomyces japonicus specific protein/dubious
SJAG_01982		Schizosaccharomyces japonicus specific disordered protein
SJAG_00387		dubious
SJAG_04967		Schizosaccharomyces japonicus specific disordered protein
SJAG_02970		Schizosaccharomyces japonicus specific disordered protein
SJAG_01306		Schizosaccharomyces japonicus specific protein with signal peptide
SJAG_00050		Schizosaccharomyces japonicus specific protein
SJAG_01091		Schizosaccharomyces japonicus specific protein in duplicated region/dubious, shorter version of SJAG_00023
SJAG_00023		Schizosaccharomyces japonicus specific protein in duplicated region/dubious, longer version of SJAG_01091
SJAG_06187     	Schizosaccharomyces japonicus specific protein
SJAG_04786		Schizosaccharomyces japonicus specific protein with signal peptide
SJAG_00258		Schizosaccharomyces japonicus specific protein

**Comment by ValWood** [comment]
SJAG_02365		Schizosaccharomyces japonicus specific protein   
SJAG_03727		Schizosaccharomyces japonicus specific protein
SJAG_05404		Schizosaccharomyces japonicus specific protein
SJAG_05404		Schizosaccharomyces japonicus specific protein 
SJAG_04057		Schizosaccharomyces japonicus specific protein
SJAG_01273		Schizosaccharomyces japonicus specific protein
SJAG_01340		Schizosaccharomyces japonicus specific protein
SJAG_05028		Schizosaccharomyces japonicus specific protein
SJAG_02577		Schizosaccharomyces japonicus specific disordered protein
SJAG_06204		Schizosaccharomyces japonicus specific protein/dubious 
SJAG_06587		dubious
SJAG_00016		dubious
SJAG_05254		amino acid/polyamine transporter
SJAG_06596		Schizosaccharomyces japonicus specific protein
SJAG_00658		Schizosaccharomyces japonicus specific protein
SJAG_05632		Schizosaccharomyces japonicus specific protein
SJAG_05201		armadillo fold, proteasome activator Blm10-like
SJAG_02476		mug129-like
SJAG_02365		Schizosaccharomyces japonicus specific protein

**Comment by ValWood** [comment]
SJAG_03304		conserved fungal family
SJAG_03639		Schizosaccharomyces japonicus specific protein
SJAG_05106		Schizosaccharomyces japonicus specific protein/dubious
SJAG_06643		Schizosaccharomyces japonicus specific protein/dubious
SJAG_03814		Schizosaccharomyces japonicas specific protein
SJAG_06627		Schizosaccharomyces japonicus specific protein
SJAG_01828		Schizosaccharomyces japonicus specific protein
SJAG_05154		mating-type homeobox protein Pi
SJAG_02987		flocculin, Flo11-related
SJAG_00388		Schizosaccharomyces japonicus specific protein
SJAG_06591		Schizosaccharomyces japonicus specific protein/dubious
SJAG_05840		dubious
SJAG_05233		dubious

[main e6abf4d] more products

**Comment by ValWood** [comment]
[main ccbd6c2] names and products

SJAG_00273              Schizosaccharomyces japonicus specific protein
SJAG_05267              dubious
SJAG_03778              dubious
SJAG_05250              Schizosaccharomyces japonicus specific protein
SJAG_00398              candidate S. pombe Sro1 ortholog, syntenic position, intrinsically disordered region
SJAG_03565              candidate S. pombe SPBC1A4.04 ortholog, syntenic position, intrinsically disordered protein
SJAG_04769              candidate S. pombe Mcp3 ortholog, syntenic position, intrinsically disordered protein
SJAG_04730              candidate S. pombe SPAC664.13 ortholog, syntenic position, intrinsically disordered protein
SJAG_05246              candidate S. pombe SPAC23C11.07 ortholog, syntenic position
SJAG_06428              candidate S. pombe SPBC12C2.01c ortholog, syntenic position
SJAG_00740              Schizosaccharomyces japonicus specific protein
SJAG_04523              candidate S. pombe SPBC28E12.04 ortholog, syntenic position
SJAG_01627              candidate S. pombe SPBP4H10.18c ortholog, syntenic position, intrinsically disordered protein
SJAG_03772              candidate S. pombe Csi1 ortholog, syntenic position, intrinsically disordered protein
SJAG_04555              candidate S. pombe Mei3 ortholog, syntenic position, intrinsically disordered protein
SJAG_02086              candidate S. pombe Mug125 ortholog, syntenic position, intrinsically disordered protein
SJAG_01730              Schizosaccharomyces japonicus specific intrinsically disordered protein
SJAG_03105              candidate S. pombe  Mug98 ortholog, syntenic position, intrinsically disordered protein
SJAG_03171              Schizosaccharomyces japonicus specific disordered protein
SJAG_01395              Schizosaccharomyces japonicus specific disordered protein
SJAG_03975              Schizosaccharomyces japonicus specific protein
SJAG_06111              Schizosaccharomyces japonicus specific protein with 2 predicted transmembrane domains
SJAG_05872              Schizosaccharomyces japonicus specific protein
SJAG_00715              dubious, SPNCRNA.1420 syntenic position
SJAG_01800              dubious, SPNCRNA.800  syntenic position
SJAG_01932              Schizosaccharomyces japonicus specific protein/dubious
SJAG_02438              dubious, 5' UTR  of Atp19

**Comment by ValWood** [comment]
[main a8a2b6c] names_and_products.tsv
SJAG_05533	Schizosaccharomyces japonicus specific protein  
SJAG_04193	cell surface protein, serine/threonine repeat
SJAG_01723	Schizosaccharomyces japonicus specific protein

**Comment by ValWood** [comment]
I have no added products to all except these, which I will continue to work through

SJAG_00389	hits tam13 jackhmmr/PSI/FASTA. Maybe tam13 is part of mug147?
SJAG_04809	No JACKHMMR/PSI. Tricky, candidate for stb3?
SJAG_06614      No jackhmmr/PSI/FASTA. join(<3047678..3047701,3047785..3047967) ->join(3047600..3047701,3047785..3047967). Search with long seq
SJAG_06638	matPc ? ask snezka 
SJAG_02888	No jackhmmr/PSI/FASTA. Messy
SJAG_01247	Syntenic with SPAC8C9.04, but different composition and length. || no jackhmmr/PSI/FASTA.
SJAG_06539	iss10? candidate No jackhmmr/PSI/FASTA? Half syntenic  jackhmmr&PSI from pombe only gets octo. and cryo.
SJAG_04962	Iss10? SPAC7D4.14c,  no jackhammer 	(FASTA pombe 	SP:O14269 cryo TR:S9W246 octo 	TR:S9R3P2)
SJAG_06618	No met	
SJAG_00625		
SJAG_04805		
SJAG_00535		
SJAG_01781		
SJAG_05269		
SJAG_05750

@snezhkaoliferenko  I'd like your input on SJAG_06638	matPc  - is this something to do with the mating type? if so  is the construct parts already named in some way?

**Comment by ValWood** [comment]
Final set:
SJAG_04962              candidate Iss10 ortholog 
SJAG_00389              Schizosaccharomyces specific protein Tam13/Mug147, tandem fusion
SJAG_04809              Schizosaccharomyces japonicus specific protein
SJAG_06614              Schizosaccharomyces japonicus specific protein
SJAG_06539              Schizosaccharomyces japonicus specific protein
SJAG_02888              Schizosaccharomyces japonicus specific protein
SJAG_00625              dubious
SJAG_04448              LDOC related- Zinc finger, CCHC-type
SJAG_04805              fungal WD repeat-like protein 
SJAG_00124              Schizosaccharomyces specific protein, S. pombe SPBC16G5.19 predicted on opposite strand, one is likely incorrect, similarity detected with tblastn (S. j 4008650-4008727) in the SPBC16G5.19 reading frame, unable to get a CDS. 

Apologies for the one at the end. There is a real problem that I can't figure out. I want to make sure that people are aware of the problem and I have updated the pombe product accordingly. I'm just not sure which strand is correct so this will serve as a warning for anyone looking. We. can make the product more sensible soon once I have figured it out.....
